website/js/AlleleFrequencyCharts.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+var React = require('react'),
+    BarChart = require('./BarChart'),
+    _ = require('lodash');
+
+// copied from index.js, move to utility file
```

Comment:
if it's straightforward to move to a utility file, may as well do it now before we have duplicate code